Library/Homebrew/blacklist.rb

```diff
@@ -89,5 +89,10 @@ def blacklisted? name
     A binary installer is available:
       https://www.haskell.org/platform/mac.html
     EOS
+  when 'gnome-icon-theme', 'adwaita-icon-theme' then <<-EOS.undent
+    The Adwaita icon theme (which replaces the now defunct gnome-icon-theme)
```

Comment:
I think there's an Alias available for `adwaita-icon-theme` which would need deleting as well.